source/parser.hera

```diff
@@ -422,8 +422,8 @@ ArrowFunction
 
 FatArrow
   # Ensures at least one space before arrow
```

Comment:
This comment seems inaccurate (both before and after), so perhaps should remove it (or remove the `?` from `_?`).
I *think* it makes sense to allow `<T>=>` though I'm not totally sure.

Reply:
It's "ensure" and not "require" it's mainly describing that inside the handler a `" "` is added before the `=>` if none present.

Reply:
Ohh, I see. Never mind then!

Could maybe clarify the comment by mentioning the "output". But it's OK as is.